TAF/taf/IORResolver_T.cpp

```diff
@@ -0,0 +1,331 @@
+/***************************************************************
+Copyright 2016, 2017 Defence Science and Technology Group,
+Department of Defence,
+Australian Government
+
+This file is part of LASAGNE.
+
+LASAGNE is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as
+published by the Free Software Foundation, either version 3
+of the License, or (at your option) any later version.
+
+LASAGNE is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public
+License along with LASAGNE.  If not, see <http://www.gnu.org/licenses/>.
+***************************************************************/
+#ifndef TAF_IORRESOLVER_T_CPP
+#define TAF_IORRESOLVER_T_CPP
+
+
+#include "IORResolver_T.h"
+#include <taf/ORBManager.h>
+
+
+#define IOR_FILENAME_EXTENSION ACE_TEXT(".ior");
```

Comment:
ACE_TEXT is only for ACE_TCHAR, this is used with a std::string so ACE_TEXT shouldn't be used. Maybe make this a const std::string?